src/viam/robot/client.py

```diff
@@ -66,6 +66,36 @@ class RobotClient:
 
     Note: Robots used within a context are automatically closed UNLESS created with a channel. Robots created using ``with_channel`` are
     not automatically closed.
+
+    Establish a Connection::
+
+        import asyncio
+
+        from viam.rpc.dial import DialOptions, Credentials
+        from viam.robot.client import RobotClient
+
+
+        async def connect():
+            opts = RobotClient.Options.with_api_key(
+                # Replace "<API-KEY>" (including brackets) with your machine's
+                # API key
```

Comment:
A lot of these comments have been split into 2 lines when they could be one. I think it would make for easier reading to put all of these on a single line (we bumped the line-length limit of 140 from 80, so you should have a lot of space)

Reply:
This applies to a bunch of comments which could be made easier to read